packages/@react-spectrum/button/test/Button.test.js

```diff
@@ -273,6 +273,40 @@ describe('Button', function () {
     expect(eventDown.defaultPrevented).toBeFalsy();
   });
 
+  // is pending state
+  it('displays a spinner after a short delay when isPending prop is true', function () {
+    jest.useFakeTimers();
```

Comment:
go ahead and put this in a beforeAll, I doubt it'll affect any other test in this file

if it does, then wrap this set of tests which is dependent on faketimers in a new describe block with a beforeAll which turns on fake timers, an after each which runs all timers, and an afterAll which restores us to real timers